### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dmitruz**
- [DEMO LINK](https://dmitruz.github.io/layout_catalog/)
- [TEST REPORT LINK](https://dmitruz.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Run the test so that mentors can see your tests, it's importan't for reviewing your home task!

File: src/styles/blocks/stars.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  &__stars-star {
+    width: 16px;
+    height: 16px;
+    background: url('../images/star.svg') no-repeat center;
+    margin-right: 4px;
+  }
+
+  &__stars--4 :nth-child(-n+4) {
+    background-image: url('../images/star-active.svg');
+  }
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/stars.scss:18]: please use scss nesting

**Review by Andriy-Shymkiv — APPROVED** [review]
lgtm

File: src/styles/blocks/stars.scss (hunk 1 above)
Comment (dmitruz) [reply src/styles/blocks/stars.scss]: fix it

**Pull request closed without merging**